docs/resources/Webhook.md

```diff
@@ -46,7 +46,7 @@ Create a new webhook. Returns a [webhook](#DOCS_WEBHOOK/webhook-object) object o
 | Field | Type | Description |
 |-------|------|-------------|
 | name | string | name of the webhook (2-32 characters) |
-| avatar | [avatar data](#DOCS_USER/avatar-data) string | base64 128x128 jpeg image for the default webhook avatar |
+| avatar | [avatar data](#DOCS_USER/avatar-data) string | image for the default webhook avatar (minimum size is 128x128) |
```

Comment:
there's actually not a minimum.. i would just drop the size information for consistency with the user avatar description (these are for all intents and purposes equivalent)